.tx/config

```diff
@@ -2,127 +2,52 @@
 host = https://www.transifex.com
 type = PO
 
-[odoo-12-doc.accounting]
-file_filter = locale/<lang>/LC_MESSAGES/accounting.po
-source_file = locale/sources/accounting.pot
-source_lang = en
-
 [odoo-12-doc.applications]
 file_filter = locale/<lang>/LC_MESSAGES/applications.po
 source_file = locale/sources/applications.pot
 source_lang = en
 
-[odoo-12-doc.crm]
-file_filter = locale/<lang>/LC_MESSAGES/crm.po
-source_file = locale/sources/crm.pot
-source_lang = en
-
-[odoo-12-doc.db_management]
-file_filter = locale/<lang>/LC_MESSAGES/db_management.po
-source_file = locale/sources/db_management.pot
-source_lang = en
-
-[odoo-12-doc.discuss]
-file_filter = locale/<lang>/LC_MESSAGES/discuss.po
-source_file = locale/sources/discuss.pot
-source_lang = en
-
-[odoo-12-doc.ecommerce]
-file_filter = locale/<lang>/LC_MESSAGES/ecommerce.po
-source_file = locale/sources/ecommerce.pot
-source_lang = en
-
-[odoo-12-doc.expenses]
-file_filter = locale/<lang>/LC_MESSAGES/expenses.po
-source_file = locale/sources/expenses.pot
+[odoo-12-doc.finance]
+file_filter = locale/<lang>/LC_MESSAGES/finance.po
+source_file = locale/sources/finance.pot
```

Comment:
No way to give transifex some sort of pattern for this? Seems like a pain in the ass to maintain and keep sync'd.

Reply:
Yep, generated by hand, super fun. Sphinx has an [option `update-txconfig-resources`](https://sphinx-sr-rs.readthedocs.io/sr/latest/usage/advanced/intl.html#using-transifex-service-for-team-translation) but wasn't working well. For odoo/odoo, I have [a script](https://github.com/mart-e/odoo-scripts/blob/master/transifex/export-translations2.py#L72) to generate it.
Hopefully, this file content is not supposed to change inside a version.